readthedocs/templates/base.html

```diff
@@ -103,8 +103,20 @@
     <div id="content">
       <div class="wrapper">
 
+        {% comment %}
+          ``user_notifications`` comes from a Context Processor.
+          We need to use a CustomUser to have access to "user.notifications"
+          See https://docs.djangoproject.com/en/4.2/ref/settings/#std-setting-AUTH_USER_MODEL
+        {% endcomment %}
+        {% for notification in user_notifications %}
+        <p class="build-failure">
+            {{ notification.get_message.get_rendered_body|safe }}
+        </p>
+        {% endfor %}
+
         <!-- BEGIN notify -->
         {% block notify %}
+          {# TODO: migrate these notifications to new system #}
```

Comment:
There is no different plan for regular Django message like "Successfully logged out" and similar ones.

The "migration" required here is due that we were using `django-messages-extends` to add extra and custom site notifications and we are not using this pattern anymore since they are now `Notification` objects.

Note that "Close notification" button that marks the notification as read has to be migrated to the new system.

Reply:
I opened an issue to continue talking about this at https://github.com/readthedocs/readthedocs.org/issues/10988

Reply:
> The "migration" required here is due that we were using django-messages-extends to add extra and custom site notifications and we are not using this pattern anymore since they are now Notification objects.

Yeah, I guess that is my confusion. The notifications remaining should be using Django messages and won't need to be ported. These will be mostly form confirmation and other temporary messages that don't require dismissal. This template block is already accurate then, but the conditionals around `message.pk` will never be truthy again and can be removed.